OptimizelySDK/OptimizelyUserContext.cs

```diff
@@ -46,6 +46,8 @@ public OptimizelyUserContext(Optimizely optimizely, string userId, UserAttribute
             UserId = userId;
         }
 
+        private OptimizelyUserContext Clone() => new OptimizelyUserContext(Optimizely, UserId, Attributes, ErrorHandler, Logger);
```

Comment:
change "Attributes" to "getAttributes()", which should be mutex protected while copying.

Reply:
Also rename it to Copy.